cart.go

```diff
@@ -51,6 +50,10 @@ type build struct {
 	StopTime string `json:"stop_time"`
 }
 
+type artifactItems struct {
```

Comment:
This doesn’t need to be a named type, it could be declared inline with `var items` below.

Reply:
Thanks for the review, fixed!